Test/FileGetContentsLoaderTest.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/*
+ * (c) Markus Lanthaler <mail@markus-lanthaler.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ML\JsonLD\Test;
+
+use ML\IRI\IRI;
+use ML\JsonLD\FileGetContentsLoader;
+
+/**
+ * Test the parsing of a JSON-LD document into a Document.
+ */
+class FileGetContentsLoaderTest extends \PHPUnit_Framework_TestCase
+{
+
+    protected $iri;
+
+    protected $loader;
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->iri = new IRI('https://www.google.com');
```

Comment:
Even though I work at Google these days I'd prefer to use example.com here and elsewhere :-)

Reply:
Heh! I'd not considered that. I've switched to foobar.com and fizzbuzz.net 😆